### PR TITLE
 Completed ticket 9, errors have already been handled

### PR DESCRIPTION
**Pull request by Hammamsgit**
I've created endpoint for ticket number 9 sorted by descending order, invalid path error already tested previously.

### CODE REVIEW

**Review by maddierosecodes — CHANGES_REQUESTED** [review]
Please could you stick to using either promises or async await? :)

File: __tests__/app.test.js [hunk 1]
```diff
@@ -127,4 +127,34 @@ describe("app", () => {
         });
     });
   });
```
Comment (popatre) [inline __tests__/app.test.js:163]: Cant actually comment on your PATCH tests above, as they're not on this pull request, but you're missing some tests for that. 
- TEST - What if the wrong data value is passed in with inc_votes - i.e. a string. 
- TEST - What is the key is spelt wrong?
- TEST - extra keys on the object?
-

**Review by popatre — CHANGES_REQUESTED** [review]

File: __tests__/app.test.js [hunk 2]
```diff
@@ -101,12 +113,32 @@ describe("app", () => {
...
+      const updateVote = { inc_votest: 1 };
+      return request(app)
+        .patch("/api/articles/3")
+        .send(updateVote)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("invalid input");
+        });
+    });
+    test("status: 400, responds with invalid input when multiple keys input", () => {
```
Comment (maddierosecodes) [inline __tests__/app.test.js:126]: just make sure the increase_votes key isn't misspelled in this test to ensure that it is failing because of the additional keys and not just due to there being no relevant key present.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -101,12 +113,32 @@ describe("app", () => {
...
+      return request(app)
+        .patch("/api/articles/3")
+        .send(updateVote)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("invalid input");
+        });
+    });
+    test("status: 400, responds with when given wrong format inputted ", () => {
       return request(app)
```
Comment (maddierosecodes) [inline __tests__/app.test.js:138]: Send an empty object for this test and reword it to say 'when body is empty'

**Review by maddierosecodes — CHANGES_REQUESTED** [review]
Just a couple changes

**Review by dav3rid — APPROVED** [review]
